packages/tailwindcss/preflight.css

```diff
@@ -189,6 +189,7 @@ textarea,
   font: inherit; /* 1 */
   font-feature-settings: inherit; /* 1 */
   font-variation-settings: inherit; /* 1 */
+  letter-spacing: inherit; /* 1 */
```

Comment:
The big difference compared to the v3 implementation is that the selector includes `::file-selector-button`. But I tested this in various browsers and I don't see a difference so I think this good to merge 👍